swift/aperture/main.swift

```diff
@@ -22,12 +13,11 @@ struct Options: Decodable {
 }
 
 func record() throws {
-  let json = arguments.first!.data(using: .utf8)!
-  let options = try JSONDecoder().decode(Options.self, from: json)
+  let options: Options = try CLI.arguments.first!.jsonDecoded()
```

Comment:
@LarsJK Any option on whether to do the above or `let options = try CLI.arguments.first!.jsonDecoded() as Options`?

Reply:
No preference.. Would probably do it the same way 👍